tests/unit/test_issue.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from pydantic.error_wrappers import ValidationError
+
+from bugjira.issue import BugzillaIssue, JiraIssue
+
+
+def test_good_bz_keys(good_bz_keys):
+    """
+    GIVEN a set of well-formatted bz keys
+    WHEN we create a BugzillaIssue with each key
+    THEN no ValidationError is raised and the issue key matches the input key
+    """
+    for key in good_bz_keys:
+        issue = BugzillaIssue(key=key)
+        assert issue.key == key
+
+
+def test_bad_bz_keys(bad_bz_keys):
+    """
+    GIVEN a set of poorly-formatted bz keys
+    WHEN we create a BugzillaIssue with each key
+    THEN a ValidationError is raised
+    """
+    for key in bad_bz_keys:
+        with pytest.raises(ValidationError):
```

Comment:
I didn't do that here because the error message is not something we define; we just get it from pydantic. My thought (insofar as I thought about it) was that I'd rather not hard code an expectation about pydantic's phrasing of the error message. They could conceivably change what the message says in a future release. So I just check to see that the ValidationError is raised.